src/model/beneficial.owner.statement.model.ts

```diff
@@ -1,4 +1,5 @@
 export const BeneficialOwnerStatementKey = "beneficial_owners_statement";
+export const UpdateBeneficialOwnerStatementKey = "update_beneficial_owners_statement";
```

Comment:
We can re-use the existing BeneficialOwnerStatementKey and don't need to add a new one to the model.

Reply:
Sorry if you got an email to the one below I was going up the page and didn't see this one 😆 

This has now been deleted